### PR TITLE
Use the word 'transform' more consistently on DESeq2 steps

### LINKED ISSUE #220: RNA-seq modules: Be more clear about normalization vs transformation when using DESeq2 functions
Origin: https://github.com/AlexsLemonade/refinebio-examples/pull/216#discussion_r490204903

This is pertinent to any modules where we have used DESeq2 transformations.

### PR DESCRIPTION
**Pull request by cansavvy**
## Purpose:
Closes #220 

## Strategy
I looked through the RNA-seq examples and looked for the sections where we applied DESeq2 transformations. I then searched and replaced those sentences that only mentioned normalization to mention transformation and link to the 00-rnaseq.html transformation section for more info. 

I also found one `TODO` for a link to the rna-seq clustering heatmap link that I fulfilled. 

And then of course, I ran snakemake on the docker container for re-rendering. 

## Concerns/Questions for reviewers:

Should I drop the word normalize entirely from these sections (I have not because I think people are more familiar with that word, so I kept it but added "transformation).  So now everything says `normalize and transform`.

### CODE REVIEW

File: 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd [hunk 1]
```diff
@@ -270,10 +270,10 @@ genes_to_keep <- rowSums(counts(dds)) >= 10
...
-## Perform DESeq2 normalization
+## Perform DESeq2 normalization and transformation
 
-We are going to use the `vst()` function from the `DESeq2` package to normalize the data.
-TODO: Add a note/link to section in documentation on why we use `vst()` vs `rlog()` and `normTransform()`.
+We are going to use the `vst()` function from the `DESeq2` package to normalize and transform the data.
+For more information about these transformation methods, [see here](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#deseq2-transformation-methods). 
 
 ```{r}
 # Normalize the data in the `DESeqDataSet` object using the `vst()` function from the `DESEq2` R package
```
Comment (cbethell) [inline 03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd:279]:
```suggestion
# Normalize and transform the data in the `DESeqDataSet` object using the `vst()` function from the `DESEq2` R package
```

We should probably propagate this change to the comments as well.

File: 03-rnaseq/dimension-reduction_rnaseq_02_umap.Rmd [hunk 2]
```diff
@@ -287,13 +287,13 @@ genes_to_keep <- rowSums(counts(dds)) >= 10
...
+## Perform DESeq2 normalization and transformation
 
-We are going to use the `vst()` function from the `DESeq2` package to normalize the data.
-TODO: Add a note/link to section in documentation on why we use `vst()` vs `rlog()` and `normTransform()`.
+We are going to use the `vst()` function from the `DESeq2` package to normalize and transform the data.
+For more information about these transformation methods, [see here](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#deseq2-transformation-methods). 
 
 ```{r}
-# Normalize the data in the `DESeqDataSet` object using the `vst()` function from the `DESEq2` R package
+# Normalize the data in the `DESeqDataSet` object using the `vst()` function from the `DESeq2` R package
```
Comment (cbethell) [inline 03-rnaseq/dimension-reduction_rnaseq_02_umap.Rmd:296]:
```suggestion
# Normalize and transform the data in the `DESeqDataSet` object using the `vst()` function from the `DESeq2` R package
```

**Review by cbethell — APPROVED** [review]
Looks good to me @cansavvy! Just one suggestion to propagate the wording to the comments in the code chunk, otherwise 🚀

**Review by jaclyn-taroni — APPROVED** [review]
Seems fine!

**Pull request merged**